schemas/com.health-union/nexus_user_context/jsonschema/1-0-0

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Health-Union/iglu-schemas/master/schemas/com.health-union/nexus_user_context/jsonschema/1-0-0",
+  "description": "Schema for Nexus snowplow user context",
+  "title": "nexus_user_context",
+  "self": {
+    "vendor": "com.health-union",
+    "name": "nexus_user_context",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "snowplow_user_id": {
+      "type": "string"
+    },
+    "google_analytics_id": {
+      "type": "string"
+    },
+    "wordpress_user_id": {
+      "type": "string"
+    },
+    "facebook_pixel_user_id": {
+      "type": "string"
+    },
+    "qualtrics_user_id": {
+      "type": "string"
+    },
+    "liveramp_user_id": {
+      "type": "string"
+    },
+    "krux_user_id": {
+      "type": "string"
+    },
+    "krux_user_segments": {
+      "type": "string"
+    },
+    "pendo_user_id": {
+      "type": "string"
+    },
+    "pendo_user_segments": {
+      "type": "string"
+    },
+    "user_login_status": {
+      "type": "string"
+    },
+    "user_role": {
+      "type": "string"
+    }  
+  },
+  "required": [
+    "snowplow_user_id",
+    "google_analytics_id",
+    "wordpress_user_id",
+    "facebook_pixel_user_id",
+    "qualtrics_user_id",
```

Comment:
Many of these fields are also not yet implemented. Tickets were created in the nexus data epic: https://health-union.atlassian.net/browse/NEXUS-836 but are still open.

These few will definitely not be implemented by launch: Pendo, facebook, liveramp, qualtrics - Krux is a maybe, depending on whether the data aligns with what we need to send to DFP (I believe it does)

Reply:
Gotcha, removed those from required list.